docs/az_test_writing.py

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
This file is just an example of how **I** expect to write music. It's not a definite guideline